### ISSUE TITLE
Shuffling the data between epochs

### ISSUE THREAD

**Issue by gtrevnenski** [AlexanderKroll/ProSmith#6]
I see that the Dataloaders have the shuffle parameter set to False and this is done because the way data loading is handled currently only allows sequential data reads. However, training the algorithm every epoch on data points in the same order is generally a bad practice. Is there another reason for that besides the convenience in terms of data loading and do you expect an increase in accuracy if the algorithm allowed for shuffling during training?

**Comment by AlexanderKroll** [comment]
Thanks for the explanation, everything you said is correct. Since the files with the ESM embeddings for all proteins are so large, we cannot load them all at once during model training, but have to load them successively. I am not changing the order of these larger files (which could also be implemented), just within these larger protein chunks. Since these are large chunks of the total dataset, I don't expect there to be much difference from completely shuffling the dataset, but of course I don't know for sure. Maybe it helps a little bit to implement more data shuffling.
